core/src/main/java/hudson/cli/DeleteBuildsCommand.java

```diff
@@ -38,7 +37,7 @@
  * @author Kohsuke Kawaguchi
  */
 @Extension
-public class DeleteBuildsCommand extends AbstractBuildRangeCommand {
+public class DeleteBuildsCommand extends JobRangeCommand {
```

Comment:
> CLI command classes are not considered APIs

By whom? No explicit documentation about that :(
But yes, Restricted would help